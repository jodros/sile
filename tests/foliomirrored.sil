\begin{document}
\set[parameter=folio.style, value=mirror]
\eject
\end{document}