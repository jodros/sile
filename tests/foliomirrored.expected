Set paper size 	595.275597	841.8897729
Begin page
Mx 	560.8194
My 	780.6177
Set font 	Gentium Plus;10;400;;normal;;;LTR
T	20 w=4.6924	(1)
New page
Mx 	29.7638
My 	780.5689
T	21 w=4.6924	(2)
End page
Finish
